src/utils/mediaObserve.ts

```diff
@@ -1,70 +0,0 @@
-type Media = 'isMobile' | 'isTablet' | 'isPC' | 'isDark'
-export type MediaMap = Partial<Record<Media, boolean>>
-export type MediaQueryMap = Partial<Record<Media, string>>
-
-const mediaMap: MediaQueryMap = {
-  isDark: '(prefers-color-scheme: dark)'
-}
-
-type SubscribeFunc = (screens: MediaMap) => void
-
-let medias = {}
-const subscribers: SubscribeFunc[] = []
-
-const mediaObserve = {
-  matchHandlers: {},
-  dispatch(newMedias: MediaMap) {
-    medias = newMedias
-    subscribers.forEach(sub => {
-      sub(medias)
-    })
-  },
-  subscribe(fn: SubscribeFunc) {
-    if (subscribers.length === 0) {
-      this.register()
-    }
-    subscribers.push(fn)
-    fn(medias)
-
-    return () => {
-      const index = subscribers.indexOf(fn)
-      if (index > -1) {
-        subscribers.splice(index, 1)
-      }
-      if (subscribers.length === 0) this.unregister()
-    }
-  },
-  register() {
-    Object.keys(mediaMap).forEach((m: string) => {
-      // @ts-ignore
-      const media = mediaMap[m as Media]!
-      const mql = window.matchMedia(media)
-      const listener = ({ matches }: { matches: boolean }) => {
-        this.dispatch({
-          ...medias,
-          [m]: matches
-        })
-      }
-      mql.addListener(listener)
```

Comment:
TS is saying this is deprecated so I removed this file. It is used in one place (see `updater`) and looks like it is used to control dark mode. I don't think we need it.